common/app/views/support/HtmlCleaner.scala

```diff
@@ -296,7 +296,7 @@ class TweetCleaner(content: Content, amp: Boolean) extends HtmlCleaner {
             element.empty()
             element.tagName("amp-twitter")
             element.attr("data-tweetId", elem.id)
-            element.attr("data-​c​ards", "hidden")
+            element.attr("data-cards", "hidden")
```

Comment:
hard to review 😝

Reply:
Looks like there are 2 unicode characters wrapping the c

element.attr("data-<U+200B>c<U+200B>ards", "hidden")

Reply:
Woo, sneakiest of them all – ‘ZERO WIDTH SPACE’. Good spot!